server/controllers/v1/post.ts

```diff
@@ -314,9 +339,36 @@ export default class PostController {
       };
     }
 
+    if (tags) {
+      // could not get adding (multiple) tags to work any other way
+      const tagObjs = await Promise.all(
+        tags.map(async (text) => {
+          return this.setupTag(text.trim());
+        })
+      );
+      await post.addTags(tagObjs);
```

Comment:
The sql would be something like:
```sql
INSERT INTO Tags (text) VALUES ('firstTag', 'secondTag', 'thirdTag');
```

Alternatively, https://sequelize.org/v5/manual/instances.html#working-in-bulk--creating--updating-and-destroying-multiple-rows-at-once-

Reply:
I looked at bulk create, but is there a way to send an array through it? It seems like you have to actually pass the specific items being created. I could use .map() with the raw insert query, but I don't think it'll work with bulkCreate().  We aren't actually inserting the text into tags here, that's done in setupTag. Did you want me to change the query in setupTag()?